src/styles/theme.js

```diff
@@ -0,0 +1,12 @@
+export const theme = {
+    colors: {
+        Font1: "#FFFFFF",
+        Box: "#EEE7EB",
+        Secondary: "#E1C9D8",
+        Primary: "#360C25",
+        Accent: "#A05042",
+        Font2: "#111010",
+        BtnHover: "#D0B5B0",
+        BtnActive: "#C68362"
```

Comment:
Not PascalCase in javascript variables. camelCase is the thing. Only use PascalCase for components